### PR TITLE
Fixes experimental ordnance launchers.

### PR DESCRIPTION
**Pull request by RemervKeberb**
## About The Pull Request

Prevents Experimental ordnance launchers from destroying their projectiles as soon as you fire them. This was caused by them having a burst size of 2, meaning two would shoot out and instantly collide unless you got very lucky with time dilation or the spread of the ordnance.
## Why It's Good For The Game

Having an upgrade be actively detrimental isn't really ideal.

## Changelog
:cl:
fix: Experimental fighter ordnance launchers no longer destroy projectiles they fire.
/:cl:

### CODE REVIEW

**Comment by RemervKeberb** [comment]
You can still easily shoot your own missiles down if you spam click but time dilation can affect a lot of factors determining how fast you can fire them out so I think it's best to just leave it up to the pilots to balance the speed of their shots depending on the circumstance.

**Comment by Karmic-Skink** [comment]
Was this ever mentioned in a gitissue?

**Comment by RemervKeberb** [comment]
I think I started writing one up at like 5 am when I discovered the issue weeks ago and then fell asleep.

File: nsv13/code/modules/overmap/fighters/_fighters.dm
Comment (Karmic-Skink) [inline nsv13/code/modules/overmap/fighters/_fighters.dm:1418]: Unrelated to the reported issue.

**Review by Karmic-Skink — CHANGES_REQUESTED** [review]

File: nsv13/code/modules/overmap/fighters/_fighters.dm
Comment (RemervKeberb) [reply nsv13/code/modules/overmap/fighters/_fighters.dm:1418]: Ah, forgot about that. The standard one currently holds 5 so the upgraded one is redundant. Meant to make them two separate PR's

**Review by Bokkiewokkie — APPROVED** [review]

**Comment by DeltaFire15** [comment]
Related problem *might* be fixed by #1672, please hold off on merging this unless it for some reason does not help fighter torps (which is entirely possible considering projectile code)

**Comment by DeltaFire15** [comment]
Update after localtesting with my changes, launched on a fighter and spammed the secondary with the T3 missile launcher, it worked fine.

**Comment by Bokkiewokkie** [comment]
> Update after localtesting with my changes, launched on a fighter and spammed the secondary with the T3 missile launcher, it worked fine.

Untestmerged because of this, we'll likely just keep the double-shot then

**Comment by RemervKeberb** [comment]
I've yet to test it myself but if the issue is fixed then perfect!

**Comment by Karmic-Skink** [comment]
I believe this has been resolved correctly, message me if this isn't the case.

**Pull request closed without merging**
